Base/etc/SystemServer.ini

```diff
@@ -141,3 +141,11 @@ StdIO=/dev/tty1
 Environment=TERM=xterm
 KeepAlive=1
 BootModes=text
+
+[CppLanguageServer]
+Socket=/tmp/portal/cpp
```

Comment:
If you're emphasizing that this is just the first of language servers (I'd anticipate a JS language server in the future, amirite?), why not put it in `/tmp/portal/language/cpp`?

Reply:
I dunno about js, but you're gonna get a shell language server whether you want it or not :P